core/src/main/java/com/googlecode/psiprobe/TomcatContainer.java

```diff
@@ -83,19 +83,19 @@
     String getServletFileNameForJsp(Context context, String jspName);
 
     List getApplicationFilterMaps(Context context);
-    
+
     boolean getAvailable(Context context);
-    
+
     public void addContextResource(Context context, List resourceList,
-			boolean contextBound);
-    
+                                   boolean contextBound);

```

Comment:
This probably shouldn't be wrapped at all, but indenting wrapped parameter lists to be aligned means that changing the method name length has a "blast radius" (as the Google style guide puts it) since you have to modify other lines.